app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -35,6 +36,22 @@ def tenant=(name)
       attrs[:tenant] = name
     end
 
+    def allow_ext_net?
+      !(attrs[:allow_ext_net] == 0)
+    end
+
+    def allow_ext_net
```

Comment:
Can you change these methods to use the full name, 'allow_external_network'? It's easier to understand for someone unfamiliar with the OpenStack API